blueocean-dashboard/src/main/js/components/karaoke/components/InputStep.jsx

```diff
@@ -149,6 +154,7 @@ const { object, shape } = PropTypes;
 InputStep.propTypes = {
     step: shape().isRequired,
     classicInputUrl: object,
+    classicInputUrl: PropTypes.string,
```

Comment:
that is duplicated @NicuPascu

Reply:
thanks for noticing, I'll make a new PR removing the duplicate code, thanks